packages/@react-spectrum/dnd/src/useDragAndDrop.ts

```diff
@@ -65,7 +65,12 @@ export interface DragAndDropOptions extends Omit<DraggableCollectionProps, 'prev
    * A function that returns the items being dragged. If not specified, we assume that the collection is not draggable.
    * @default () => []
    */
-  getItems?: (keys: Set<Key>) => DragItem[],
+  getItems?: (
+    /** The set of keys that can be affected by the drag (e.g. the current selection). */
```

Comment:
I think the individual ones are useful for autofill. But they still don't appear in our docs. Are you seeing them somewhere? maybe I'm just looking in the wrong place
![Screenshot 2025-06-26 at 9 34 21 am](https://github.com/user-attachments/assets/63f052c0-bb6c-47d8-a3ea-5589307c6b17)
https://reactspectrum.blob.core.windows.net/reactspectrum/a00898e3b00bd2f2ad3954a86d8e8df764b7623b/docs/react-aria/useDraggableCollection.html

Reply:
Just in VSCode, not seeing them in the docs

Reply:
Then we should probably add it to the main description body for the prop as well. I think leave in the more granular ones for VSCode though as well